pom.xml

```
@@ -10,18 +10,20 @@
 	</parent>
 	<groupId>io.jenkins.plugins</groupId>
 	<artifactId>inodes-monitor</artifactId>
-	<version>0.6-SNAPSHOT</version>
+	<version>${revision}</version>
```

This needs to be `changelist` rather than `revision`.
```suggestion
	<version>${changelist}</version>
```

---

Thanks for your time and suggestions @MarkEWaite and @NotMyFault 
the suggested changes has been added